proto/iam/authenticated_client_claims.proto

```diff
@@ -7,10 +7,11 @@ option go_package = "iam";
 import "iam/staff_member_indicator.proto";
 import "iam/tenancy_indicator.proto";
 import "iam/client_instance_indicator.proto";
+import "primitive/session_context.proto";
 
 message AuthenticatedClientClaims {
```

Comment:
This shouldn't be a proto message.

It currently is a proto message because we used to transmit ACCs across rabbitmq, before we figured out that transmitting JWTs was a better idea. Having this as a proto message defeats the entire purpose of this object -- it's designed specifically to convey that "you shouldn't pass this all over the place, it should only be used very locally, in call paths where the callee trusts the caller has properly converted the token into a set of claims". 

I would ask that you get rid of this now, but it might turn out to take a few minutes, especially if we have to rework a few transmitted ACC scenarios to use tokens. I filed a chore to fix this later (https://www.pivotaltracker.com/story/show/135541575)